### PR TITLE
Merge d16-2 into xcode11

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

File: mk/xamarin.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ifdef ENABLE_XAMARIN
-NEEDED_MACCORE_VERSION := 62a239f3f89573dd68c120993cd1dfeb166f8c9c
+NEEDED_MACCORE_VERSION := 12ecfd8249cab8d3867a779e5ce633f97bb3825b
```
Comment (rolfbjarne) [inline mk/xamarin.mk]: This hash is from maccore's `d16-2` branch, it should be the tip of maccore's `xcode11` branch.

**Comment by spouliot** [comment]
`java.lang.NullPointerException`

**Comment by spouliot** [comment]
build

**Comment by spouliot** [comment]
mix of 
* msbuild path issue https://github.com/mono/mono/issues/14875
* NSData/https https://github.com/xamarin/maccore/issues/1725
* HE0038 https://github.com/xamarin/maccore/issues/581

**Pull request merged**